packages/forms-web-app/src/dynamic-forms/middleware/redirect-to-unanswered-question.js

```diff
@@ -0,0 +1,25 @@
+const { getJourney } = require('../journey-factory');
+const config = require('../../config');
+
+module.exports = () => (req, res) => {
+	const journeyResponse = res.locals.journeyResponse;
+	const journey = getJourney(journeyResponse);
+
+	for (const section of journey.sections) {
+		for (const question of section.questions) {
+			const answer = journey.response?.answers[question.fieldName];
+			if (!answer || answer === '' || answer === undefined) {
+				let redirectUrl;
+
+				if (section.segment === config.dynamicForms.DEFAULT_SECTION) {
```

Comment:
did having a single section with no url not work then?

Reply:
I couldnt get it to work, not without having to make changes throughout the journey/ question classes, the controllers and validators as the section is pretty embedded into how everything works. 

This is the route i was originally going down until we had the discussion last week. I wasnt removing the coupling though, just adding an if check more or less anywhere the section/segment was used and putting null as the argument for the section. It definitely wasnt optimal.

This currently uses a default section which allows all the existing functions to work as they are. The section is just not used in the url as there is only the one.